xrtmodels/response.go

```diff
@@ -8,12 +8,13 @@ import (
 )
 
 const (
-	// https://github.com/IOTechSystems/xrt/blob/master/include/devsdk/devsdk.h#L27
+	// https://docs.iotechsys.com/edge-xrt21/mqtt-management/mqtt-management.html#general-result-format
 	XrtSdkStatusOk               = 0
 	XrtSdkStatusNotFound         = 1
 	XrtSdkStatusNotSupported     = 2
 	XrtSdkStatusInvalidOperation = 3
 	XrtSdkStatusAlreadyExists    = 7
+	XrtSdkStatusServerError      = 20 // custom error code because XRT haven't defined the Server Error
```

Comment:
use 500 to prevent the future conflict of xrt error code

Reply:
Updated.